### PR TITLE
Adds Portuguese translations

### LINKED ISSUE #17530: Add Portuguese translations
### Description

Add Portuguese (Portugal) translations besides existing Portuguese (Brazil)

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by froque**
Closes #17530

### CODE REVIEW

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_pt.properties [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+actionNotDefined=Nenhuma ação definida
+continue=Continuar
+refreshPage=Atualizar página
+refresh=Atualizar
+done=Pronto
+cancel=Cancelar
+remove=Remover
+update=Atualizar
+loadingMessage=Carregando consola da conta...
+unknownUser=Anónimo
```
Comment (trixpan) [inline themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_pt.properties:18]: use unicode across the board

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_pt.properties [hunk 2]
```diff
@@ -0,0 +1,177 @@
+# encoding: UTF-8
+# Put new messages for Account Console Here
+# Feel free to use any existing messages from the base theme
+pageNotFound=Página não encontrada
+forbidden=Proibido
+needAccessRights=Não tem as permissões de acesso para esta solicitação. Entre em contato com um administrador.
```
Comment (trixpan) [inline themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_pt.properties:6]: use unicode across the board

**Review by trixpan — CHANGES_REQUESTED** [review]
you seem to be mixing unicode and non-unicode messages.

IMNSHO you should stick to using the unicode notation across all files.

**Comment by froque** [comment]
@trixpan I thought I was using UTF-8 already. 

```shell
❯ find themes/src/ -name messages_pt.properties| xargs file -i 
themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_pt.properties: text/plain; charset=utf-8
themes/src/main/resources-community/theme/base/admin/messages/messages_pt.properties:          text/plain; charset=utf-8
themes/src/main/resources-community/theme/base/account/messages/messages_pt.properties:        text/plain; charset=utf-8
themes/src/main/resources-community/theme/base/email/messages/messages_pt.properties:          text/html; charset=utf-8
themes/src/main/resources-community/theme/base/login/messages/messages_pt.properties:          text/plain; charset=utf-8
❯ find themes/src/ -name messages_pt.properties| xargs isutf8
```

This seems to be similar to latest Arabic translation addition https://github.com/keycloak/keycloak/pull/17090
```
❯ find themes/src/ -name messages_ar.properties| xargs file -i 
themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_ar.properties: text/plain; charset=utf-8
themes/src/main/resources-community/theme/base/admin/messages/messages_ar.properties:          text/plain; charset=utf-8
themes/src/main/resources-community/theme/base/account/messages/messages_ar.properties:        text/plain; charset=utf-8
themes/src/main/resources-community/theme/base/email/messages/messages_ar.properties:          text/html; charset=utf-8
themes/src/main/resources-community/theme/base/login/messages/messages_ar.properties:          text/plain; charset=utf-8
❯ find themes/src/ -name messages_ar.properties| xargs isutf8
```

Am I missing something ?

**Comment by codespearhead** [comment]
@froque 

Unlike all the other edited files in your commit, [this file](https://github.com/keycloak/keycloak/blob/5190a02ace37dfc5737ba2f5bd2f85cf1606d532/themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_pt.properties) isn't using Unicode escape sequences.

- OK
https://github.com/keycloak/keycloak/blob/5190a02ace37dfc5737ba2f5bd2f85cf1606d532/themes/src/main/resources-community/theme/base/login/messages/messages_pt_BR.properties#L7

- Not OK
https://github.com/keycloak/keycloak/blob/5190a02ace37dfc5737ba2f5bd2f85cf1606d532/themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_pt.properties#L18

The Arabic translation is an exception since everything would have to be represented as Unicode escape sequences.

I'm willing to collaborate on this PR btw (I'm Brazilian).

**Comment by jonkoops** [comment]
Closing this as it has massive conflicts that are not likely to be resolved. All our property files are read as UTF-8 now, so there is no need to use any escape sequences.

**Pull request closed without merging**